getMongoData/getMongoData.js

```diff
@@ -304,7 +304,15 @@ function printDataInfo(isMongoS) {
                               function(){return db.getSiblingDB(mydb.name).getCollection(col).getIndexes()}, section);
                     if (col != "system.users") {
                         printInfo('Sample document',
-                                  function(){return db.getSiblingDB(mydb.name).getCollection(col).findOne()}, section);
+                                  function(){
+					var lastValCursor = db.getSiblingDB(mydb.name).getCollection(col).find().sort({'$natural': -1}).limit(1);
+					if (lastValCursor.hasNext()) {
+						return lastValCursor.next()
+					}
+					else {
+						return {'warning': 'no sample document found'};
```

Comment:
I know it's extremely unlikely, but this is potentially ambiguous.  At the very least, it should be `$warning` rather than `warning`.  However, I would strongly prefer just returning `null` in this case, which is completely unambiguous and the same as what `findOne` does.